integ-tests/src/cucumberTest/java/cucumber/steps/dataservice/DataServiceSteps.java

```diff
@@ -143,7 +145,7 @@ public DataServiceSteps() {
           final EnvironmentId environmentId = getEnvironmentIdFromCreatedEnvironment();
           dataServiceWrapper.updateEnvironment(
               inputCreator.updateEnvironmentRequestWithNewCluster(
-                  environmentId.getEnvironmentName(), newCluster));
+                  environmentId.getEnvironmentName(), inputCreator.prefixName(newCluster)));
```

Comment:
This is not in the scope of this PR, but should we change this update into "I update the created environment with new task definition" ? For the purpose of testing, updating the cluster name should be fine. But I'm not sure in reality will there be a use case of updating the cluster.

Reply:
Definitlely. Updating the cluster name actually makes no sense; we shouldn't support moving Environments between Clusters, or Weird Things May Happen.